azure/template.json

```
@@ -11,9 +11,13 @@
         "serviceName": {
             "type": "string"
         },
+        "keyvaultCertificateName": {
+            "type": "string",
+            "defaultValue": ""
+        },
         "configNames": {
             "type": "string",
-            "defaultValue": "__SolutionBaseName__"
+            "defaultValue": "__SolutionName__"
```

```suggestion
            "defaultValue": "SFA.DAS.ApprenticeService.Web"
```
If no config needed in das-employer-config though, this app setting and version aren't needed